tools/pack.sh

```diff
@@ -4,10 +4,7 @@ repo_root=$(cd $(dirname $0)/.. && pwd)
 cd ${repo_root}
 
 echo "Cleaning dist directories..."
-rm -rf dist/
-rm -rf packages/cdk8s/dist
-rm -rf packages/cdk8s-plus-17/dist
-rm -rf packages/cdk8s-cli/dist
+rm -rf **/dist
```

Comment:
Convert `pack.sh` to a projen Task and derive these `rm -rf` execs from the project structure.

What if a `node_module` includes a `dist` directory?